test/runtests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# runtests.sh
+
+# create bin directory if it doesn't exist
+if [ ! -d "../bin" ]
+then
+    mkdir ../bin
+fi
+
+# delete output from previous run
+if [ ! -d "../actual.txt"]
```

Comment:
There should be a space between the last `"` and `]`. `Bash` is smart enough to figure this out but there will be a warning because it's not the proper syntax.